sample/src/androidTest/java/com/schibsted/spain/barista/sample/LonfClickIdAndTextTest.java

```diff
@@ -0,0 +1,36 @@
+package com.schibsted.spain.barista.sample;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import com.schibsted.spain.barista.sample.util.FailureHandlerValidatorRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertDisplayed;
+import static com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertNotExist;
+import static com.schibsted.spain.barista.interaction.BaristaClickInteractions.longClickOn;
+
+@RunWith(AndroidJUnit4.class)
+public class LonfClickIdAndTextTest {
```

Comment:
Typo here `LonfClickIdAndTextTest` should be `LongClickIdAndTextTest`

Reply:
Nice catch!